packages/frontpage/app/(app)/post/new/_client.tsx

```diff
@@ -49,7 +49,6 @@ export function NewPostForm() {
         <Input
           name="url"
           id={`${id}-url`}
-          type="url"
```

Comment:
Suggestion: We can probably add a regex html validation here to give the user instant feedback before submitting the form.

Reply:
How? The issue is that there is a client side validation error if the protocol is left off.

If what you are saying is we should do regex here and let the user know they need a protocol then that is fine, but that is a different fix / flow I think.

I think as a user, I should be able to share a link as `www.google.com`